test/components/personkort/PersonkortHeaderTest.tsx

```diff
@@ -56,6 +50,14 @@ describe("PersonkortHeader", () => {
     expect(screen.queryByText("Egenansatt")).not.to.exist;
   });
 
+  it("viser ikke 'Egenansatt' når isEgenansatt er true fra API, men man ikke bruker await i expecten (altså er ikke disse testene så bra)", async () => {
```

Comment:
Bør legge til await .find på noe annet som skal dukke opp før man sjekker not.to.exists her ja.